.ruff.toml

```diff
@@ -285,26 +284,26 @@ unfixable = [
 # When a general exclusion is being fixed, but it affects many subpackages, it
 # is better to fix for subpackages individually. The general exclusion should be
 # copied to these subpackage sections and fixed there.
-"astropy/__init__.py" = []
-"astropy/config/*" = []
-"astropy/constants/*" = []
-"astropy/convolution/*" = ["I001"]
-"astropy/coordinates/*" = ["I001"]
-"astropy/cosmology/*" = []
+"astropy/__init__.py" = ["C408"]
+"astropy/config/*" = ["C408"]
```

Comment:
`astropy/config` already complies with C408, so Ruff shouldn't be told to ignore this rule there.